styles.css

```diff
@@ -210,3 +208,30 @@ aside {
   border-color: #FD8078;
   color: #FD8078;
 }
+
+.selection-error-box {
+  margin-top: -15px;
+  margin-left: 88px;
+}
+
+.minutes-seconds {
+  display: flex;
+  flex-direction: row;
+}
+
+#seconds-error {
```

Comment:
Great job with these error messages, they look great and then they disappear, as they should!